docs/guides/software-lifecycle/security/secrets-detection/README.md

```diff
@@ -77,15 +75,15 @@ This layer directly scans the developer's local environment using the `detect-se
 
 #### Steps
 1. **Installation**
-   - Install the experimental version of [slim-detect-secrets](https://github.com/NASA-AMMOS/slim-detect-secrets/tree/exp).
+   - Install [detect-secrets](https://github.com/Yelp/detect-secrets).
      ```bash
-     pip install git+https://github.com/NASA-AMMOS/slim-detect-secrets.git@exp
+     pip install git+https://github.com/Yelp/detect-secrets.git
      ```
 
 2. **Scanning**
    - Scan all local files from the current directory and output results to a baseline file.
      ```bash
-     detect-secrets scan --all-files --disable-plugin AbsolutePathDetectorExperimental --exclude-files '\.secrets.*' --exclude-files '\.git*' > .secrets.baseline
+     detect-secrets scan --all-files --exclude-files '\.secrets.*' --exclude-files '\.git*' > .secrets.baseline
```

Comment:
Ditto

Reply:
Thanks again - fixed! I updated the `.pre-commit` file that also uses this command 👍